src/components/Bulma/RJSFFormFieldAdapter/RJSFFormFieldAdapter.js

```diff
@@ -68,29 +72,36 @@ export const CustomArrayFieldTemplate = props => {
         className={classNames({"inline": _get(props, 'uiSchema.items.ui:options.inline')})}
       >
         {element.children}
-
         {element.hasRemove &&
-        <button className="button is-clear array-field__item__control remove-item-button"
-                onClick={element.onDropIndexClick(element.index)}>
-          <span className="icon is-danger">
-            <SvgSymbol sym="trash-icon" viewBox='0 0 20 20' />
-          </span>
-        </button>
+          <button className={classNames(
+            "is-clear array-field__item__control remove-item-button",
```

Comment:
You could also just pass a ternary as the second arg to classNames in this case instead of an object, e.g. `!deleteLabel ? "button" : "mr-button mr-button mr-button--small"`